run-scripts/get-dkv-runtime-code.js

```diff
@@ -0,0 +1,28 @@
+const { web3 } = require("hardhat");
```

Comment:
What is this file for? Can you provide it a short introduction here?

Reply:
Looks like auto deploy a minimal test net...

Reply:
It is convenient for us to get the runtime code of DecentralizedKVDaggerHashimoto which will be set as the precompile-contract of our Geth.

Reply:
You can have a clear understanding through this part of the code https://github.com/QuarkChain/go-ethereum/blob/tm_w3q/core/vm/contracts.go#L681